src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java

```diff
@@ -184,8 +200,12 @@ private static synchronized void rebuildInitialTagInfoListWithGaps() {
    * #refreshTagList()}. Add the parse tag information to the tag information list.
    *
    * @param line EBD line string
+   * @throws JSONException if int to string enumeration JSON parse fails
+   * @throws IOException if int to string enumeration file read fails
    */
-  private static synchronized void processTagListEBDLine(String line) {
+  private static synchronized void processTagListEBDLine(String line)
+      throws IOException, JSONException {
+
     /*
```

Comment:
```suggestion
  private static synchronized void processTagListEBDLine(String line)
      throws IOException, JSONException {
    /*
```

extra blank line added -- remove